pkg/util/mount/mount_linux.go

```diff
@@ -240,13 +248,31 @@ func readProcMountsFrom(file io.Reader, out *[]MountPoint) (uint32, error) {
 func (mounter *SafeFormatAndMount) formatAndMount(source string, target string, fstype string, options []string) error {
 	options = append(options, "defaults")
 
+	// Run fsck on the disk to fix repairable issues
+	args := []string{"-a", source}
+	cmd := mounter.Runner.Command("fsck", args...)
```

Comment:
> How do you feel about this relative to mount? I would expect them to have about the same level of 'i expect this to be in path' -- thoughts?

I agree... In fact in Fedora they come from the same package. It looks like mount_linux.go is also just using mount from $PATH. I think nsenter_mount is a special case really since it is trying to avoid path and deliberately use mount from `/rootfs`